tuna/rocmlir/rocmlir_tables.py

```diff
@@ -176,8 +178,7 @@ def __repr__(self) -> str:
       'conv_stride_w': '-v',
       'dilation_h': '-l',
       'dilation_w': '-j',
-      'group_size': '-g',
-      'data_type': '-t',
+      'group_size': '-m conv -g',  # Hack to get "-m conv -g 1"
```

Comment:
Our original conv configs always have that "-m conv" which has no direct equivalent here.